examples/causal_language_modeling/peft_prompt_tuning_refactor.py

```diff
@@ -0,0 +1,146 @@
+import argparse
+from accelerate.utils import write_basic_config
+from transformers import AutoModelForCausalLM, AutoTokenizer, default_data_collator, get_linear_schedule_with_warmup
+from peft import get_peft_config, get_peft_model, PromptTuningInit, PromptTuningConfig, TaskType, PeftType
+import torch
+from datasets import load_dataset
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+
+device = "cuda"
```

Comment:
Here the device is hard-coded, but later on `examples/conditional_generation/mlflow_peft_adalora_seq2seq.py` there is a proper check for `cuda` availability. Please be consistent and add the trivial check